spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/onnx/Constant.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.intel.analytics.bigdl.nn.onnx
+
+import scala.reflect.ClassTag
+import com.intel.analytics.bigdl.nn.abstractnn.TensorModule
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+
+
+/**
+ * A layer contains a constant tensor.
+ * It forwards the value of constant tensor to next layers.
+ *
+ * @param value constant tensor
+ * @param `classTag$T`
+ * @param ev
+ * @tparam T The numeric type in this module parameters
+ */
+class Constant[T: ClassTag](
```

Comment:
why not reusing reuse `com.intel.analytics.bigdl.nn.tf.Const` ?

Reply:
As metioned earlier, the Constant layer takes **no** input.
com.intel.analytics.bigdl.nn.tf.Const takes input, and it extends Operation, which does not allow backforward.

Reply:
Do you mean the Const operation in onnx could have preceding layer/ops?